manifests/listen.pp

```diff
@@ -11,7 +11,7 @@
     $debug_footer = ''
   }
   riemann::config::fragment { "listen_${title}":
-    content => [ "${type}-server", $options ],
```

Comment:
Same here...

Reply:
Disable the checks that enforce Puppet style or do we put each item on a single line?

Reply:
go for each on a single line. What I would like to keep is spaces when array has only one element.